### ISSUE TITLE
Installation instructions for Windows don't work

### ISSUE THREAD

**Issue by lebeg** [apache/incubator-mxnet#11100]
In the [Installing MXNet in Windows](https://mxnet.incubator.apache.org/install/windows_setup.html) instructions there is a paragraph:

> Installing the Prebuilt Package on Windows
> MXNet provides a prebuilt package for Windows. The prebuilt package includes the MXNet library, all of the dependent third-party libraries, a sample C++ solution for Visual Studio, and the Python installation script. To install the prebuilt package:
>
> Download the latest prebuilt package from the **Releases** tab of MXNet.
Unpack the package into a folder, with an appropriate name, such as D:\MXNet.
Open the folder, and install the package by double-clicking **setupenv.cmd**. This sets up all of the environment variables required by MXNet.
Test the installation by opening the provided sample C++ Visual Studio solution and building it.
This produces a library called libmxnet.dll.

**Issues**
* There are no Prebuilt Packages published at the mentioned [Releases](https://github.com/apache/incubator-mxnet/releases) page
* The mentioned `setupenv.cmd` can nowhere be found

**Comment by anirudhacharya** [comment]
@sandeep-krishnamurthy please label this - "Windows", "Installation"

**Comment by jacob081** [comment]
I also cant not find setupenv.cmd.

**Comment by kalyc** [comment]
@lebeg thanks for opening this issue. You could refer to discussions on https://discuss.mxnet.io to learn more about windows installation.

**Comment by yajiedesign** [comment]
https://github.com/yajiedesign/mxnet/releases base package

**Comment by lebeg** [comment]
Sure, thanks @yajiedesign. The documentation needs to be updated anyway.

**Comment by vishaalkapoor** [comment]
@larroy @lebeg does the PR address the this issue completely or do pre-built images still need to be added to the release page?

**Comment by lebeg** [comment]
Yes, this still needs to be done. The windows build is far from being perfect.

**Comment by aaronmarkham** [comment]
@lebeg - the instructions have been updated. However, there's no mention of these pre-built binaries in the new instructions except for the R binding. Should the guide be further updated to show users how to take advantage of the pre-built binaries for a vanilla MXNet install?
@vrakesh @ankkhedia @samskalicky - adding you all for visibility.

**Comment by piyushghai** [comment]
Related PR to windows installation instructions : https://github.com/apache/incubator-mxnet/pull/12696

**Comment by larroy** [comment]
@Chancebair

**Comment by lebeg** [comment]
> Should the guide be further updated to show users how to take advantage of the pre-built binaries for a vanilla MXNet install?

Sure, the more documentation the better!

**Comment by vishaalkapoor** [comment]
What are the outstanding issues for this issue? Can we have a checklist so that it is easier to keep track of progress?

Adding a few names for visibility:  @lebeg @aaronmarkham @piyushghai

**Comment by aaronmarkham** [comment]
@Chancebair is this still on your plate?
